go/pkg/client/cas.go

```diff
@@ -1238,34 +1238,34 @@ func (c *Client) DownloadDirectory(ctx context.Context, d digest.Digest, execRoo
 		return nil, stats, err
 	}
 
-	outStats, err := c.downloadOutputs(ctx, outputs, execRoot, cache)
+	outStats, err := c.downloadOutputs(ctx, outputs, outDir, cache)
 	stats.addFrom(outStats)
 	return outputs, stats, err
 }
 
 // DownloadActionOutputs downloads the output files and directories in the given action result. It returns the amount of downloaded bytes.
 // It returns the number of logical and real bytes downloaded, which may be different from sum
 // of sizes of the files due to dedupping and compression.
-func (c *Client) DownloadActionOutputs(ctx context.Context, resPb *repb.ActionResult, execRoot string, cache filemetadata.Cache) (*MovedBytesMetadata, error) {
+func (c *Client) DownloadActionOutputs(ctx context.Context, resPb *repb.ActionResult, outDir string, cache filemetadata.Cache) (*MovedBytesMetadata, error) {
```

Comment:
FWIW after a reread of the comments, if you do want to go through the trouble, I think `downDir` is a more appropriate name - DownloadOutputs does not necessarily download outputs to the output directory (eg it can download them to a temporary location for racing strategies)

Reply:
I think outDir is reasonable enough and I really don't want to go renaming this again :)